### ISSUE TITLE
[Feature Request]: Cross-post on Mastodon

### ISSUE THREAD

**Issue by s3nnet** [GetPublii/Publii#1668]
### Feature Description

Use a X to Mastodon bridge or post seperately on Mastodon.
Reasoning:  X is a bloody mess. Open-source 3rd party clients for X are de facto read-only by now. Not to mention the political associations with X. By and large, Mastodon seems to be a better fit for a FOSS project like Publii anyway.

**Comment by candidexmedia** [comment]
When you say cross-post, do you mean having your blog posts available to Mastodon users via the Activity Pub protocol (sort of like [Plume](https://joinplu.me/) and [Write Freely](https://writefreely.org/))?

Are you hoping to track webmentions as well?

**Comment by s3nnet** [comment]
Nope... I mean switch to mastodon (preferred), or do both (fine) , or at least mirror x posts to mastodon (there is Buffer, Zapier and whatnot). There are different getpublii mastodon accounts to be found, so I guess Publii was screwing around with it at some point.

**Comment by candidexmedia** [comment]
Ahhh, I understand what you're saying now. You're asking the Publii team to create a Mastodon account for the project, and post on there like they do on Twitter.

> There are different getpublii mastodon accounts to be found, so I guess Publii was screwing around with it at some point.

Nothing's to say that those accounts were created by the official Publii team. Unless there's a verified link and rel=me tag, there's no way of knowing for sure (related discussion on Mastodon profile verifications for Publii sites: https://github.com/GetPublii/Publii/discussions/1485)


On a tangential note: it *is* possible to cross-post your indexed Publii posts to your Mastodon account via an RSS-ActivityPub bridge called [MastoFeed](https://mastofeed.org). [BridgyFed](https://fed.brid.gy/) is also an interesting option.
